src/Controller/CRUDController.php

```diff
@@ -426,6 +426,9 @@ public function batchAction()
             $allElements = (bool) $data['all_elements'];
             $request->request->replace(array_merge($request->request->all(), $data));
         } else {
+            $request->request->set('idx', $request->get('idx', []));
```

Comment:
Now I see it, what about to move it after this line:
```php
$allElements = $request->request->getBoolean('all_elements');
```
So there is no need to call `Request::get` again and we can use `$idx` and `$allElements`.